### PR TITLE
Automatic security updates

### PR DESCRIPTION
**Pull request by meyskens**
Same as the [PR](https://github.com/scaleway/image-ubuntu/pull/49) I sent for Ubuntu.

### CODE REVIEW

**Comment by moul** [comment]
LGTM

**Comment by QuentinPerez** [comment]
LGTM, Thank you @meyskens 😊

File: sid/overlay/etc/apt/apt.conf.d/50unattended-upgrades [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Unattended-Upgrade::Allowed-Origins {
+    "Debian sid-security";
```
Comment (leoetlino) [inline sid/overlay/etc/apt/apt.conf.d/50unattended-upgrades]: Are you sure sid-security exists?

Comment (meyskens) [inline sid/overlay/etc/apt/apt.conf.d/50unattended-upgrades]: That i was not sure about.

File: wheezy/overlay/etc/apt/apt.conf.d/50unattended-upgrades [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Unattended-Upgrade::Allowed-Origins {
+    "Debian wheezy-security";
```
Comment (QuentinPerez) [inline wheezy/overlay/etc/apt/apt.conf.d/50unattended-upgrades]:
```
"origin=Debian,archive=stable,label=Debian-Security";
"origin=Debian,archive=oldstable,label=Debian-Security";
```

File: jessie/overlay/etc/apt/apt.conf.d/50unattended-upgrades [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Unattended-Upgrade::Allowed-Origins {
+    "Debian jessie-security";
```
Comment (QuentinPerez) [inline jessie/overlay/etc/apt/apt.conf.d/50unattended-upgrades]:
```
"o=Debian,n=jessie,l=Debian-Security";
```

File: sid/overlay/etc/apt/apt.conf.d/50unattended-upgrades (hunk 1 above)
Comment (QuentinPerez) [inline sid/overlay/etc/apt/apt.conf.d/50unattended-upgrades]: Quote https://wiki.debian.org/DebianUnstable

```Does sid have security updates?
Not in the same sense that stable does. If the maintainer of a package fixes a security bug and uploads the package, it'll go into sid by the normal means. If the maintainer doesn't do that, then it won't. The security team only covers stable (and possibly testing... there's a pending issue for that case).

Sid users are strongly urged to subscribe to the Debian security announce mailing list. And while you're at it, you should also be on the Debian devel announce list and Debian devel list.
```

**Pull request merged**